internal/resource/config/localidentity/identityprofile/local_identity_profile_resource.go

```diff
@@ -965,6 +980,72 @@ func (r *localIdentityProfileResource) ValidateConfig(ctx context.Context, req r
 				"If registration_config.captcha_enabled is set to true, then registration_config.captcha_provider_ref must be configured. If registration_config.captcha_enabled is false, then registration_config.captcha_provider_ref must not be configured.")
 		}
 	}
+
+	if internaltypes.IsDefined(model.FieldConfig) && internaltypes.IsDefined(model.FieldConfig.Attributes()["fields"]) {
+		fieldConfigFields := model.FieldConfig.Attributes()["fields"].(types.Set).Elements()
+		if len(fieldConfigFields) > 0 {
+			for _, field := range fieldConfigFields {
+				fieldAttrs := field.(types.Object).Attributes()
+				fieldType := fieldAttrs["type"].(types.String).ValueString()
+				hasDefault := internaltypes.IsDefined(fieldAttrs["default_value"])
+				hasOptions := internaltypes.IsDefined(fieldAttrs["options"])
+
+				switch fieldType {
+				case "CHECKBOX":
+					// check to make sure options isn't set
+					if hasOptions {
+						resp.Diagnostics.AddError("Invalid attribute combination",
```

Comment:
It would be good to use AddAttributeError in these so that the output from terraform is easier to parse